tensorflow/core/kernels/conv_grad_filter_ops.cc

```diff
@@ -595,6 +596,12 @@ class Conv2DSlowBackpropFilterOp : public OpKernel {
     if (filter_shape.num_elements() == 0) {
       return;
     }
+    if (input.shape().num_elements() == 0) {
```

Comment:
Could you also add this to conv_input_filter_ops.cc?
Also, could you make the comments for this consistent by saying something like: if there is nothing to cmpute, return empty tensors as the output.

Reply:
You mean `conv_grad_filter_ops.cc`? It has handled this correctly.
I'll add comments.